### DISCUSSION TITLE
React Native 开发心得分享 | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#283] [General]
# React Native 开发心得分享 | 愧怍

愧怍的个人博客

https://kuizuo.cn/blog/react-native-develop-experience/

**Comment by ksdaylight** [comment]
破局：🍞开面包店(*/ω＼*)

**Reply by kuizuo** [reply]
回去继承家业是吧(^-^*)/

**Comment by makerjackie** [comment]
写的好棒！

**Reply by kuizuo** [reply]
写的其实有点匆忙，排版和目录结构不是很好，我现在的心思都不是在创作文章，而且在写项目🥲

**Reply by makerjackie** [reply]
已经很好了哈哈哈，另外我是 google 搜索 -> 腾讯云转发的文章 -> 到你的博客的

**Comment by PeterChen1997** [comment]
很赞👍 样式选型那块很有收获

**Comment by zyw-Wayne** [comment]
很赞，最近在学RN，这篇文章帮助很大
